src/main/java/org/shanerx/tradeshop/bitrade/BiShopCreateEventListener.java

```diff
@@ -44,51 +45,50 @@ public BiShopCreateEventListener(TradeShop instance) {
 	}
 	
 	@SuppressWarnings("deprecation")
-	@EventHandler
+	@EventHandler(priority = EventPriority.HIGH)
 	public void onSignChange(SignChangeEvent event) throws InterruptedException {
-		//	BlockState state = event.getBlock().getState();
 		Player player = event.getPlayer();
 		Sign s = (Sign) event.getBlock().getState();
-		if (!(event.getLine(0).equalsIgnoreCase("[BiTrade]"))) {
+		
+		if (!event.getLine(0).equalsIgnoreCase("[BiTrade]")) {
```

Comment:
thats for the creation signs(so if a player doesnt cap them) its like that in all the creation files, but only for the first line